assets/app/views/_pods.html

```diff
@@ -5,9 +5,10 @@
 <div class="component-block pod-container">
   <div class="animate-repeat pod osc-object" ng-repeat="pod in pods track by (pod | uid)" kind="Pod" resource="pod"
       ng-class="(pod | isTroubledPod) ? 'pod-warning' : ('pod-' + pod.status.phase.toLowerCase())">
-    <div class="pod-text">
+    <div class="pod-text" ng-switch="pod.status.phase">
       <strong class="pod-status-label">{{pod.status.phase}}</strong>
-      <div ng-if="pod.status.podIP">{{pod.status.podIP}}</div><div ng-if="!pod.status.podIP">allocating IP...</div>
+      <div ng-switch-when="Pending">pulling images...</div>
```

Comment:
ooo... if we have a startTime, the kubelet saw the pod, otherwise, it's scheduling...